docs/kyma/04-01-overview.md

```diff
@@ -28,17 +28,20 @@ To make the local development process easier, we introduced the **Kyma Lite** co
 | `service-catalog-addons` | ✅ | ✅ |
 | `nats-streaming` | ✅ | ✅ |
 
+> \* In order to install velero, it needs to be set up first. Please follow the Kyma Backups setup instructions below.
+
 ## Installation guides
 
 Follow these installation guides to install Kyma locally or on a cluster:
 
 - [Install Kyma locally](#installation-install-kyma-locally)
 - [Install Kyma on a cluster](#installation-install-kyma-on-a-cluster)
 
-Read rest of the installation documents to learn how to:
+Read the rest of the installation documents to learn how to:
 - [Disable the selected components' installation or install them separately](#configuration-custom-component-installation)
 - [Upgrade Kyma to a new version](#installation-upgrade-kyma)
 - [Update Kyma](#installation-update-kyma)
 - [Reinstall Kyma](#installation-reinstall-kyma)
+- [Setup Kyma Backups](/components/backup)
```

Comment:
```suggestion
- [Set up Kyma backup](/components/backup#overview-overview)
```

Reply:
I do not think this is gramatically correct...
Either we say "Set up Kyma Backups" in plural so we do not need an article, or we say "Set up the kyma backup".
The current suggestion sounds strange to me .

Reply:
Fixed thanks!